### PR TITLE
Remove the need to run docker-auto-labels

### PR DESCRIPTION
**Pull request by m1yag1**

### CODE REVIEW

**Review by therealmarv — CHANGES_REQUESTED** [review]
Remove docker-auto-labels instructions fully from https://github.com/openstax/output-producer-service/blob/master/docs/operations/updating_the_stack.rst too?

Are the new instructions on how to create labels missing?

**Comment by m1yag1** [comment]
The instructions on how to create the labels are there in the changes.

File: docs/operations/setting_up_the_swarm.rst [hunk 1]
```diff
@@ -161,13 +161,15 @@ Create Main Traefik Service
 
    export NODE_ID=$(docker info -f '{{.Swarm.NodeID}}')
 
-**STEP 3: Add node label to the master node in the swarm**
+**STEP 3: Add labels to the master node in the swarm**
 
 .. code-block:: bash
 
    docker node update --label-add proxy=true $NODE_ID
+   docker node update --label-add app-db-data=true $NODE_ID
```
Comment (m1yag1) [inline docs/operations/setting_up_the_swarm.rst:169]: @therealmarv this adds the app-db-data=true label to the first node in the swarm.

File: docker-compose.deploy.volumes-placement.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ services:
       placement:
         constraints:
           - node.role == manager
-          - node.labels.${STACK_NAME}.app-db-data == true
+          - node.labels.app-db-data == true
```
Comment (m1yag1) [inline docker-compose.deploy.volumes-placement.yml:10]: This sets the label for the database volume to the one set in the documentation. I've already done this in the swarm.

**Comment by m1yag1** [comment]
@therealmarv I believe I resolved the comments. I removed mention of docker-auto-labels in the docs and I also removed the documentation on creating the tunnel as that is not needed when paramiko is also installed. I've tested this out with the new CORGI CD pipeline and works great. We're able to set the ssh host and docker takes care of the rest.

**Review by therealmarv — APPROVED** [review]
Looks good now for me.

**Pull request merged**